src/components/Carousel.js

```diff
@@ -256,6 +256,10 @@ class Carousel extends Component {
             itemSize: itemSize,
             wrapperSize: isHorizontal ? itemSize * this.props.children.length : itemSize
         });
+
+        if (this.refs.thumbs) {
+            this.refs.thumbs.updateSizes();
```

Comment:
This is so `updateSizes` on Carousel triggers it for Thumbs too so everything updates nicely.